definitions/variable/electricity.yaml

```
@@ -28,3 +28,10 @@
 - Discount Rate|Electricity:
     description: Discount rate applied to electricity-related investments
     unit: '%'
+- Final Energy|Electricity [Share]:
+    description: Share of load within the specified region
+    unit: ''
```

If it's a share, it might me more intuitive to use '%' as the unit and add the range attribute to highlight that it's between 0 and 100% (not between 0 and 1).
```suggestion
    unit: '%'
    range: [ 0,100 ]
```